scripts/update_schema_6.14.0.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS `iseq_run`;
+CREATE TABLE `iseq_run` (
+  `id_run` int(10) unsigned NOT NULL COMMENT 'NPG run identifier',
+  `id_flowcell_lims` varchar(20) DEFAULT NULL COMMENT 'LIMS specific flowcell id',
+  `folder_name` varchar(64) DEFAULT NULL COMMENT 'Runfolder name',
+  PRIMARY KEY (`id_run`),
+  KEY `id_flowcell_lims` (`id_flowcell_lims`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 \
```

Comment:
Could you please add `COLLATE=utf8_unicode_ci` as in the `iseq_flowcell` table. This matters when joining on varchar columns